### ISSUE TITLE
Staging Review finding: Move alert to top of review page 

### ISSUE THREAD

**Issue by shiragoodman** [department-of-veterans-affairs/va.gov-team#135398]
_Need help? Please review [how to read a Staging Review ticket](https://depo-platform-documentation.scrollhelp.site/collaboration-cycle/anatomy-of-a-staging-review-issue-ticket). Tag `@platform-governance-team-members` on Slack if you need further assistance._

## Product Information
**Team:** GovCIO VEBT
**Product:** Authorization To Disclose Personal Information To a Third Party - Education Benefits
**Feature:** VA Form 22-10278

## Findings details

**VA.gov Experience Standard - issue:** [User encounters design components or patterns that are inconsistent or confusing.](https://design.va.gov/about/experience-standards/#user-encounters-consistent-clear-design-components)
**VA.gov Experience Standard - category:** [Consistency](https://design.va.gov/about/experience-standards/#category-consistency)

**Launch-blocking:** No 
**Design System review:** No
**Collab Cycle Reviewer:** @allison0034 (Design), @jasonday (Accessibility), @erinrwhite (IA)

### Description

The zip code was somehow entered as “1234” on when you go to the review page you are unable to submit and get this alert error. The alert is currently placed below the review content. Per our guidance, the standard Alert (in all of its variations) should be placed directly below the intro text, near the top of the page, unless it is applicable to a specific section.

<details><summary>Link, screenshot or steps to recreate</summary>

<img width="736" height="758" alt="Review error" src="https://github.com/user-attachments/assets/4523a58b-d1cc-448e-8f42-1bc753b94392" />

</details>

## Recommended action

Since this alert is generic and related to the whole page, move the alert to the top of the page.

## References

- **Accessibility Defect Severity:** N/A
- **WCAG Success Criteria:** N/A
- **Modality:** N/A
- **Design System Component:** [Alert component](https://design.va.gov/components/alert)
- **Design System Pattern or Template:** N/A
- **Design System Foundation:** N/A
- **Context:** N/A
- **VADS Guidance:** N/A

---

### Next Steps for the VFS Team

_How did this touchpoint go? [Give feedback on the Collaboration Cycle](https://ows.io/qs/o3jkwoe) at any time._
- [ ] **Questions?** For the most timely response, comment on Slack in your team channel tagging `@platform-governance-team-members` with any questions or to get help validating the issue.
- [ ] **Ticket ownership:** If you believe that this issue is out of scope, not your team's responsibility, or a Design System issue; or if you believe this finding should not be fixed; comment and tag `@platform-governance-team-members` on your team channel in Slack to provide an explanation and who you believe is responsible. The Governance team will follow up.
- [ ] **Close this ticket** when the issue has been resolved or validated by your Product Owner.

**Comment by tbaker1026** [comment]
@VishnhavAshok We have not made any progress on this one - if you want to close it - we have the relationship linked to this ticket where the work will be done.

**Comment by shiragoodman** [comment]
sorry, going to interject. Please don't close the ticket @VishnhavAshok - thank you!
